src/javascript/jsdoc.ts

```diff
@@ -74,6 +74,11 @@ function parsePolymerElement(tag: doctrine.Tag): Tag {
   return {tag: tag.title, type: null, name: tag.description, description: null};
 }
 
+// @polymerMixin [name]
+function parsePolymerMixin(tag: doctrine.Tag): Tag {
+  return {tag: tag.title, type: null, name: tag.description, description: null};
```

Comment:
Please add a TODO to the code here, file a bug, or add a skipped test, just to help us keep track of it.

Ah, nm, found the TODO later on.